### PR TITLE
Add recommendations to arde5730

### PR DESCRIPTION
**Pull request by arde5730**
closes #23

### CODE REVIEW

File: githubarde5730_TrashCans.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
 ### Places to Visit
-
+- Balboa Park
+- Trash eating places
 ### Places to Avoid
-
+Nooo
 ### Fun Facts
+Not at all
```
Comment (abhikabra8811) [inline githubarde5730_TrashCans.md:15]: That itself is a fun fact 😉

**Comment by abhikabra8811** [comment]
😄

File: githubarde5730_TrashCans.md (hunk 1 above)
Comment (arde5730) [reply githubarde5730_TrashCans.md:15]: @abhikabra8811 fun fun

**Review by abhikabra8811 — APPROVED** [review]
👍

**Pull request merged**